.docker-hub/frontend/Dockerfile

```diff
@@ -23,6 +27,12 @@ RUN apk --no-cache add --virtual native-deps g++ make python3 git && npm ci && a
 COPY frontend .
 COPY --from=pdf-build-stage /pdf/dist ./src/pdf
 COPY --from=pdf-build-stage /pdf/src/assets/fonts/Inter ./src/assets/fonts/Inter
+
+ENV SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN
```

Comment:
We don't want our sentry project and access token to be present in the docker images, because the docker images should be reusable for other ecamp instances than ours.

As far as I understand, this access data is theoretically only needed during the build to upload the source maps to our sentry account. Did you check that these secrets aren't available in the built bundle files, nor in the env variables inside the container, nor in the image layers which we upload publicly to docker hub?

In case we cannot guarantee that the secrets are not present in the image, I guess we could create and upload the source maps in a separate github action instead.

Reply:
I also thought about that.

the env variables are added in a separate build steps, they are not available in the layers we push to github.
I also checked that:
```
$ docker run --rm  bacluc/ecamp3-frontend:bacluc-devel env
HOSTNAME=62db29ae9495
SHLVL=1
HOME=/root
PKG_RELEASE=1
NGINX_VERSION=1.24.0
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
NJS_VERSION=0.7.12
PWD=/app
```
I also grepped for the token in both images, they are not in the filesystem generated by the topmost layer.

Maybe we can try this later if our build fails because sentry is not available,
but i would see that as a further improvement